src/Codeception/Util/Shared/InheritedAsserts.php

```diff
@@ -40,7 +40,13 @@ protected function assertArrayNotHasKey($key, $array, string $message = '')
      */
     protected function assertClassHasAttribute(string $attributeName, string $className, string $message = '')
     {
-        Assert::assertClassHasAttribute($attributeName, $className, $message);
+        trigger_error(__FUNCTION__ . ' is removed from PHPUnit since PHPUnit 10', E_USER_DEPRECATED);
```

Comment:
"**was** removed from PHPUnit since version 10.0", since it already happened, right?